props/saber_BC_buttons.h

```diff
@@ -386,31 +1134,317 @@ EFFECT(volmax);     // for maximum volume reached
 #warning The faston.wav sound will be replaced with fastout.wav. If you have a good reason to keep faston.wav as is, please post at https://crucible.hubbe.net/
 EFFECT(faston);     // for EFFECT_FAST_ON. Being replaced by fastout.wav, which is already defined in the main OS.
 #endif
-EFFECT(blstbgn);    // for Begin Multi-Blast
-EFFECT(blstend);    // for End Multi-Blast
 EFFECT(push);       // for Force Push gesture
-EFFECT(monosfx);    // for Monophonically played sounds (iceblade, seismic charge etc...)
-EFFECT(swap);       // for standalone triggering EffectSequence<>
+EFFECT(tr);         // for EFFECT_TRANSITION_SOUND, use with User Effects.
+EFFECT2(trloop, trloop);  // have YET to play with this
 EFFECT(mute);       // Notification before muted ignition to avoid confusion.
+EFFECT(mzoom);      // for Spam Blast enter/exit
+
+#ifndef BC_ENABLE_OS_MENU
+template<class SPEC>
+struct BCVolumeMode : public SPEC::MenuBase {
+  const int max_volume_ = VOLUME;
+  const int min_volume_ = VOLUME * 0.10;
+  float initial_volume_ = 0.0;
+  int initial_percentage_ = 0;
+  int last_volume_ = 0;
+  int current_volume_ = 0;
+  int percentage_ = 0;
+
+int steps_per_revolution() override {
```

Comment:
indentation (should be same as the line above)

Reply:
thanks.